github-arc/cluster/upstream/ClusterRoleBinding.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: manager-role
+subjects:
+  - kind: ServiceAccount
+    name: default
```

Comment:
This should be something other then `default` - powerful permissions granted to default SA.

Reply:
what can it be set to?